### ISSUE TITLE
Updated onboarding - Welcome screens.

### ISSUE THREAD

**Issue by iccub** [brave/brave-ios#4372]
See `Mobile Onboarding 2021 Revision` doc in shared drive.

- [ ] Implement welcome screens
- [ ] Implement shields intro popup
- [ ] Implement 3dot menu intro popup

**Comment by GeetaSarvadnya** [comment]
Verification **PASSED** on `iPad 7th Generation version iOS 15.1` running  Brave version `1.33 (21.11.30.21)`

- Logged https://github.com/brave/brave-ios/issues/4603
- Logged https://github.com/brave/brave-ios/issues/4607
- Logged https://github.com/brave/brave-ios/issues/4608
- Logged https://github.com/brave/brave-ios/issues/4610
- Logged https://github.com/brave/brave-ios/issues/4619


### US Region

<details>
<summary>Case 1: Implement welcome screens_New user_PASSED</summary>

- Confirmed onboarding welcome screens are implemented as per in the Figma doc https://www.figma.com/file/irDJRP35FUwsRu60Wp9fM6/iOS-onboarding?node-id=469%3A5764
- Confirmed content text in the 2nd welcome screen `You're just a step away from the best privacy online. Ready?` and the title `Privacy made easy` is as per the Figma spec
- Click on `Let's go` on the 2nd welcome screen opens up the `set as default browser` slide
- Confirmed content text displayed in default browser slide as per the Figma spec
- Confirmed click on `Set as default` button opens up the device default browser app setting screen
- Confirmed user can set brave as default browser via onboarding modal
- Confirmed click on `Not now` opens up next `You're ready to browse!` slide
- Confirmed content text displayed under `You're ready to browse!` slide is as per the Figma spec
- Confirmed  3 `Websites` displayed under `You're ready to browse!` slide vary based on the region
- Confirmed tapping `Skip` on any of the welcome screens lands the user on the NTP
- Confirmed users can click on any one of the 3 websites they wish and websites open up in the NTP
- Confirmed user can enter their own website they wish in the `Enter a website` search box and the website opens up in NTP
- Confirmed kill the app during onboarding and relaunch app will lands the user on the NTP

**_US Region - Dark theme_** 

Welcome 1 | Welcome 2 | Set brave as default | Go to site | Site opened
-------|--------|-------|---------|-----
![IMG_0070](https://user-images.githubusercontent.com/38657976/144401527-67cdbd51-96e1-46a0-a125-a4f173b285ea.PNG) | ![IMG_0066](https://user-images.githubusercontent.com/38657976/144401592-6e542402-1ff1-4f0b-8df9-133336ac5b29.PNG) | ![IMG_0067](https://user-images.githubusercontent.com/38657976/144401684-2d43a43d-09d6-4a44-a8ca-32411f0ccdcc.PNG) | ![IMG_0068](https://user-images.githubusercontent.com/38657976/144401768-43ed286a-d01d-430a-8ae1-b009052b98e7.PNG) | ![IMG_0069](https://user-images.githubusercontent.com/38657976/144402541-d5cdb5fe-dfc2-41c2-9960-900aa79bdf66.PNG)

**_US Region - Light theme_** 

Welcome 1 | Welcome 2 | Set brave as default | Go to site | Site opened
-------|--------|-------|---------|-------
![IMG_0071](https://user-images.githubusercontent.com/38657976/144411762-d9322a03-cb1b-4703-b8bf-2d667ef3daf8.PNG) | ![IMG_0072](https://user-images.githubusercontent.com/38657976/144411797-32cbe1d2-971f-400d-b92a-850339f76627.PNG) | ![IMG_0073](https://user-images.githubusercontent.com/38657976/144411831-8ed3b468-21a7-4255-bab7-9b4865d334c9.PNG) | ![IMG_0074](https://user-images.githubusercontent.com/38657976/144411896-46760fc3-d632-4667-b6be-ea54522b2c57.PNG) | ![IMG_0075](https://user-images.githubusercontent.com/38657976/144411951-945426bb-713a-4d54-bc26-405ba30ae55c.PNG)
</details>

<details>
<summary>Case 2: Implement welcome screens_Existing user_PASSED</summary>

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and then upgrade the profile to `1.33 (21.11.30.21)`  and confirm that the welcome onboarding flow isn't shown.
- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.11.30.21)`  and confirm that the welcome onboarding flow isn't shown.
</details>

<details>

<summary>Case 3: Implement shields intro popup_New user_PASSED</summary>

Light theme | Dark theme
-------------|-------------
![IMG_0078 (1)](https://user-images.githubusercontent.com/38657976/144449910-79a60797-b33b-4989-9c7f-d39089923f78.PNG) | ![IMG_0077](https://user-images.githubusercontent.com/38657976/144449939-ef5eb1de-46c9-4d61-8014-88dadab59e5c.PNG)

- Clean profile, click on Skip in any welcome screen and open any site via NTP presents the `shields onboarding intro` on the website
- Clean profile `1.33.x`, go through the onboarding flow and select any one website among the 3 in the `You're ready to browse!` slide and confirm that `shields onboarding intro` is shown on the website
- Clean profile `1.33.x`, go through the onboarding flow and enter any website in the `Enter a website` search box in the `You're ready to browse!` slide, and confirm that `shields onboarding intro` is shown on the website
- Clean profile `1.33.x`, skip onboarding flow, enter buzzfed.com in the URL bar and confirm `shields onboarding intro` is shown once on a site. Open the 2nd site and ensure `shields onboarding intro` isn't shown 
- Clean profile `1.33.x`, skip onboarding flow and tap on top tiles in NTP and confirm `shields onboarding intro` is shown on the website
- Clean profile `1.33.x`, skip onboarding flow and long press on any top tiles and open the site in `Private tab` and ensure `shields onboarding intro` is shown on the website and tap outside the bubble and ensure shield onboarding message is dismissed
</details>

<details>
<summary>Case 4: Implement shields intro popup_Existing user-PASSED</summary>

**_User NOT interacted with a shield in 1.32.x_**

-  Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and then upgrade the profile to `1.33 (21.12.6.21)` and tap on any top tiles in NTP and ensure `shields onboarding intro` isn't shown on the website. 

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and then upgrade the profile to `1.33 (21.12.6.21)` and open a new NTP and enter buzzfeed.com in the URL bar and ensure `shields onboarding intro` isn't shown on the website. 

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and then upgrade the profile to `1.33 (21.12.6.21)`  and long press on any Top tile and open the site in a Private tab and ensure `shields onboarding intro` isn't shown on the website

**_User interacted with a shield in 1.32.x_**

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.6.21)` and confirmed `shields onboarding intro` isn't shown on the active tab

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.6.21)` open an NTP and tap on any top tiles and ensure `shields onboarding intro` is not shown on the website 

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.6.21)` and open a new NTP and enter buzzfeed.com in the URL bar and ensure `shields onboarding intro` is not shown on the website 

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.6.21)`  and long press on any Top tile and open the site in a Private tab and ensure `shields onboarding intro` is not shown on the website
</details>

<details>
<summary>Case 5: Implement NTP intro message popup_New user_PASSED</summary>

- Confirmed NTP intro message is shown when a user visits NTP for the 1st time, open 2nd NTP, and ensured the NTP intro message is not shown
- Confirmed content text shown in `NTP intro popup message` is as per the Figma spec
- Clean profile 1.33.x, click on the `Skip` button on any welcome slide opens up NTP with intro message
- Clean profile 1.33.x, click on the `Not now` button in the `Privacy made easy` slide and click on any site in the next slide opens up the website, open a new NTP will show the intro popup intro message
- Clean profile 1.33.x, go through the welcome onboarding flow and click on any website in the `You're ready to browser` slide opens up the website, open a new NTP will show the intro popup intro message
- Clean profile 1.33.x, go through the welcome onboarding flow and enter any website in the `You're ready to browser` slide search box opens up NTP with intro message. Encountered https://github.com/brave/brave-ios/issues/4619

Light theme | Dark theme
-------|--------
![IMG_0083](https://user-images.githubusercontent.com/38657976/144563582-78211a23-ab47-4f64-9d4a-facb41f57f41.PNG) | ![IMG_0084](https://user-images.githubusercontent.com/38657976/144563534-68db07fa-cf6d-4fa1-ada7-1ef1ca049824.PNG)

- Confirmed Tap anywhere outside the intro message popup will dismiss popup in NTP
- Confirmed browser intro more menu `All set!` popup message is shown on NTP after dismissing the NTP intro popup message
- Confirmed content text of the `All set!` popup is as per the Figma spec
- Confirmed Tap anywhere outside the `All set!` popup will dismiss the more menu popup message
- Click on the 3 dot menu opens up the settings page

Light theme | Dark theme
-------|--------
![IMG_0086](https://user-images.githubusercontent.com/38657976/144574354-ee011565-f254-44ec-8a19-397bebc1d10c.PNG) | ![IMG_0085](https://user-images.githubusercontent.com/38657976/144574373-322971d9-6695-49a5-9704-f24d33c10926.PNG)

</details>

<details>
<summary>Case 6: Implement NTP intro message popup_Existing user_PASSED</summary>


- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.11.30.21)`. Open an NTP and ensure the `NTP intro onboarding` message is not shown

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and upgrade the profile to `1.33 (21.11.30.21)` and ensure `NTP intro onboarding` message is not shown in the NTP 

</details>

### JP Region

<details>
<summary>Onboarding flow_New user_PASSED</summary>

**Welcome screens**

- Confirmed welcome onboarding flow is shown correctly for JP region
- Confirmed content text of each slide is as per the Figma spec
- Confirmed click on `Skip` button in any slide opens up the NTP
- Confirmed enter any website via last slide opens up the entered website correctly in a NTP

**_Light theme_**
Example | Example | Example | Example 
---------|---------|---------|---------
![IMG_0151](https://user-images.githubusercontent.com/38657976/144990944-77ff47d2-4906-408c-b897-3bc87225d184.PNG) | ![IMG_0139](https://user-images.githubusercontent.com/38657976/144990973-fecac5ac-551e-496f-a866-1e528ea59f2e.PNG) | ![IMG_0140](https://user-images.githubusercontent.com/38657976/144991003-9a06fe77-6f49-438a-a984-b268a36fc54f.PNG) | ![IMG_0141](https://user-images.githubusercontent.com/38657976/144991017-3acc490d-03ff-4a9c-bd8c-ff3afd8a1899.PNG)

**_Dark theme_**
Example | Example | Example | Example 
---------|---------|---------|---------
![IMG_0143](https://user-images.githubusercontent.com/38657976/144990410-20844622-6afe-4a17-976c-a27850dd0b94.PNG) | ![IMG_0144](https://user-images.githubusercontent.com/38657976/144990387-9c650a5c-2c79-4a5f-a492-05b6f63aca0c.PNG) | ![IMG_0145](https://user-images.githubusercontent.com/38657976/144990347-9ab65d21-f56c-4fb4-81d5-f9aafdf34298.PNG) | ![IMG_0146](https://user-images.githubusercontent.com/38657976/144990300-c14e5680-1fc5-4467-a2c0-9038acac82b1.PNG)


**Shield intro message**
- Confirmed shield intro message is shown when any website is opened via NTP
- Confirmed shield intro message popup is shown only once
- Open any website via the last slide and ensure the shield intro message is shown on the opened website
- Tap on any top tiles in NTP and ensure shield intro popup message is shown
- Open an NTP and enter any website and ensure shield intro popup message is shown

Light theme | Dark theme
--------|---------
![IMG_0142](https://user-images.githubusercontent.com/38657976/144988441-bc1e5fca-0768-4991-88f5-3bf5f366ccce.PNG) | ![IMG_0148](https://user-images.githubusercontent.com/38657976/144989616-2fa18732-68a8-4fc0-a4dc-24a3a398c6c9.PNG)


**NTP intro message**
- Confirmed NTP intro message is shown in NTP when welcome onboarding is skipped
- Confirmed NTP intro message is shown only once
- Enter any website via the last slide and ensure shield intro message is shown on the website, ensured an NTP intro message is shown when an NTP is opened
- Tap on any site in the last slide and ensure shield intro message is shown on the website, ensured an NTP intro message is shown when an NTP is opened

**_Light theme_**

Example | Example 
--------|---------
![IMG_0135](https://user-images.githubusercontent.com/38657976/144988871-d5540240-5533-49ce-86c0-53fa5f20e57e.PNG) | ![IMG_0136](https://user-images.githubusercontent.com/38657976/144988908-d6c94fd6-24a1-4507-877d-b9a4ef2c2796.PNG)

**_Dark theme_**

Example | Example 
--------|---------
![IMG_0149](https://user-images.githubusercontent.com/38657976/144989938-107548ff-cb9f-4a8f-98c5-4c1d27fb876b.PNG) | ![IMG_0150](https://user-images.githubusercontent.com/38657976/144989990-cd04f17c-20bd-45d6-a645-8cf3c384bfd5.PNG)

</details>

<details>
<summary>Onboarding flow_Existing user_PASSED</summary>

**_Onboarding flow_**
- Install the app store version `1.32.3 (21.11.17.19)` and then upgrade the profile to `1.33 (21.12.6.21)`  and confirm that the welcome onboarding flow isn't shown.
- Install the app store version `1.32.3 (21.11.17.19)` and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.6.21)`  and confirm that the welcome onboarding flow isn't shown.

**_Shield intro_**

-  Install the app store version `1.32.3 (21.11.17.19)` and then upgrade the profile to `1.33 (21.11.30.21)`  and ensure shield intro message is not shown on the active tab, tap on any top tiles in NTP and ensure `shields onboarding intro` isn't shown on the website. 

- Install the app store version `1.32.3 (21.11.17.19)` and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.6.21)` open an NTP and tap on any top tiles and ensure `shields onboarding intro` is not shown on the website 

- Install the app store version `1.32.3 (21.11.17.19)` and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.6.21)` and open a new NTP and enter buzzfeed.com in the URL bar and ensure `shields onboarding intro` is not shown on the website 

- Install the app store version `1.32.3 (21.11.17.19)` and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.6.21)` and long press on any Top tile and open the site in a Private tab and ensure `shields onboarding intro` is not shown on the website

**_NTP intro_**

- Install the app store version `1.32.3 (21.11.17.19)` and upgrade the profile to `1.33 (21.12.6.21)` and ensure the `NTP intro onboarding` message is not shown in the NTP 

</details>

**Comment by GeetaSarvadnya** [comment]
Confirmed `Privacy report` implementation is not part of this release `1.33.x` got confirmation from @Brandon-T and @soner-yuksel - Please refer to the thread https://bravesoftware.slack.com/archives/C06UXF3KJ/p1638462573385900?thread_ts=1638460257.382300&cid=C06UXF3KJ

**Comment by stephendonner** [comment]
Verification `PASSED` on `iPhone 6s on iOS 14.4.2` running Brave version `1.33 (21.12.10.20)`

### US Region

<details>
<summary>Case 1: Implement welcome screens_New user - PASSED</summary>

- Confirmed onboarding welcome screens are implemented as per in the Figma doc https://www.figma.com/file/irDJRP35FUwsRu60Wp9fM6/iOS-onboarding?node-id=469%3A5764
- Confirmed content text in the 2nd welcome screen `You're just a step away from the best privacy online. Ready?` and the title `Privacy made easy` is as per the Figma spec
- Click on `Let's go` on the 2nd welcome screen opens up the `set as default browser` slide
- Confirmed content text displayed in default browser slide as per the Figma spec
- Confirmed click on `Set as default` button opens up the device default browser app setting screen
- Confirmed user can set brave as default browser via onboarding modal
- Confirmed click on `Not now` opens up next `You're ready to browse!` slide
- Confirmed content text displayed under `You're ready to browse!` slide is as per the Figma spec
- Confirmed  3 `Websites` displayed under `You're ready to browse!` slide vary based on the region
- Confirmed tapping `Skip` on any of the welcome screens lands the user on the NTP
- Confirmed users can click on any one of the 3 websites they wish and websites open up in the NTP
- Confirmed user can enter their own website they wish in the `Enter a website` search box and the website opens up in NTP
- Confirmed kill the app during onboarding and relaunch app will lands the user on the NTP

**_US Region - Dark theme_** 

Welcome 1 | Welcome 2 | Set brave as default | Go to site | Site opened
-------|--------|-------|---------|-----
![Image from iOS (6)](https://user-images.githubusercontent.com/387249/145654862-68fa822e-f491-4163-b49f-f4079b15b084.png) | ![Image from iOS (7)](https://user-images.githubusercontent.com/387249/145654870-05413c2e-f450-4264-b6d3-6f172d89ab1f.png) | ![Image from iOS (8)](https://user-images.githubusercontent.com/387249/145654873-605a0f6a-b932-4069-882b-23745864db19.png) | ![Image from iOS (9)](https://user-images.githubusercontent.com/387249/145654877-c0efe3a2-b4d1-42bf-8ea5-fff68a2dcc52.png) | ![Image from iOS (10)](https://user-images.githubusercontent.com/387249/145654880-5f0466a9-89a9-45fb-b843-426df23e598d.png)


**_US Region - Light theme_** 

Welcome 1 | Welcome 2 | Set brave as default | Go to site | Site opened
-------|--------|-------|---------|-------
![Image from iOS](https://user-images.githubusercontent.com/387249/145654984-ed5b9d8a-49fb-4cd9-848c-52c29289970a.png) | ![Image from iOS (1)](https://user-images.githubusercontent.com/387249/145654999-b21e2548-ffe7-4172-a075-e1c6691d6375.png) | ![Image from iOS (2)](https://user-images.githubusercontent.com/387249/145655005-1e451be2-e52f-48f0-99fb-f3ca76dcbd50.png) | ![Image from iOS (3)](https://user-images.githubusercontent.com/387249/145655006-d40bbe0e-cc36-4c9a-b67d-c4b53a44d579.png) | ![Image from iOS (4)](https://user-images.githubusercontent.com/387249/145655007-87213bdb-0ed9-4bd2-8768-02240c7ae50b.png)




</details>

<details>
<summary>Case 2: Implement welcome screens_Existing user - PASSED</summary>

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and then upgrade the profile to `1.33 (21.12.10.20)`  and confirm that the welcome onboarding flow isn't shown.
- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.10.20)`  and confirm that the welcome onboarding flow isn't shown.
</details>

<details>

<summary>Case 3: Implement shields intro popup_New user - PASSED</summary>

Light theme | Dark theme
-------------|-------------
![Image from iOS (5)](https://user-images.githubusercontent.com/387249/145656556-0c09c43b-ad22-458f-b654-bb44d60bd1b6.png) | ![Image from iOS (6)](https://user-images.githubusercontent.com/387249/145656557-d0dd7193-b577-4332-8cea-5a49287e2650.png)


- Clean profile, click on Skip in any welcome screen and open any site via NTP presents the `shields onboarding intro` on the website
- Clean profile `1.33.x`, go through the onboarding flow and select any one website among the 3 in the `You're ready to browse!` slide and confirm that `shields onboarding intro` is shown on the website
- Clean profile `1.33.x`, go through the onboarding flow and enter any website in the `Enter a website` search box in the `You're ready to browse!` slide, and confirm that `shields onboarding intro` is shown on the website
- Clean profile `1.33.x`, skip onboarding flow, enter `buzzfeed.com` in the URL bar and confirm `shields onboarding intro` is shown once on a site. Open the 2nd site and ensure `shields onboarding intro` isn't shown 
- Clean profile `1.33.x`, skip onboarding flow and tap on top tiles in NTP and confirm `shields onboarding intro` is shown on the website
- Clean profile `1.33.x`, skip onboarding flow and long press on any top tiles and open the site in `Private tab` and ensure `shields onboarding intro` is shown on the website and tap outside the bubble and ensure shield onboarding message is dismissed
</details>

<details>
<summary>Case 4: Implement shields intro popup_Existing user - PASSED</summary>

**_User NOT interacted with a shield in 1.32.x_**

-  Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and then upgrade the profile to `1.33 (21.12.10.20)` and tap on any top tiles in NTP and ensure `shields onboarding intro` isn't shown on the website. 

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and then upgrade the profile to `1.33 (21.12.10.20)` and open a new NTP and enter buzzfeed.com in the URL bar and ensure `shields onboarding intro` isn't shown on the website. 

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and then upgrade the profile to `1.33 (21.12.10.20)`  and long press on any Top tile and open the site in a Private tab and ensure `shields onboarding intro` isn't shown on the website

**_User interacted with a shield in 1.32.x_**

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.10.20)` and confirmed `shields onboarding intro` isn't shown on the active tab

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.10.20)` open an NTP and tap on any top tiles and ensure `shields onboarding intro` is not shown on the website 

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.10.20)` and open a new NTP and enter buzzfeed.com in the URL bar and ensure `shields onboarding intro` is not shown on the website 

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.10.20)`  and long press on any Top tile and open the site in a Private tab and ensure `shields onboarding intro` is not shown on the website
</details>

<details>
<summary>Case 5: Implement NTP intro message popup_New user - PASSED</summary>

- Confirmed NTP intro message is shown when a user visits NTP for the 1st time, open 2nd NTP, and ensured the NTP intro message is not shown
- Confirmed content text shown in `NTP intro popup message` is as per the Figma spec
- Clean profile 1.33.x, click on the `Skip` button on any welcome slide opens up NTP with intro message
- Clean profile 1.33.x, click on the `Not now` button in the `Privacy made easy` slide and click on any site in the next slide opens up the website, open a new NTP will show the intro popup intro message
- Clean profile 1.33.x, go through the welcome onboarding flow and click on any website in the `You're ready to browser` slide opens up the website, open a new NTP will show the intro popup intro message
- Clean profile 1.33.x, go through the welcome onboarding flow and enter any website in the `You're ready to browser` slide search box opens up NTP with intro message. Encountered https://github.com/brave/brave-ios/issues/4619

Light theme | Dark theme
-------|--------
![Image from iOS](https://user-images.githubusercontent.com/387249/145656884-71304627-ab27-4520-bf50-8888bc79c01b.png) | ![Image from iOS (1)](https://user-images.githubusercontent.com/387249/145656895-cf807f9e-54b3-4d53-959d-a3f20025d97c.png)



- Confirmed Tap anywhere outside the intro message popup will dismiss popup in NTP
- Confirmed browser intro more menu `All set!` popup message is shown on NTP after dismissing the NTP intro popup message
- Confirmed content text of the `All set!` popup is as per the Figma spec
- Confirmed Tap anywhere outside the `All set!` popup will dismiss the more menu popup message
- Click on the 3 dot menu opens up the settings page

Light theme | Dark theme
-------|--------
![Image from iOS (2)](https://user-images.githubusercontent.com/387249/145656915-ed45624b-9a94-4e3b-8d5b-3aee983530ff.png) | ![Image from iOS (3)](https://user-images.githubusercontent.com/387249/145656919-938c6d04-b2f4-4c91-ad6c-5a0736dc1c25.png)


</details>

<details>
<summary>Case 6: Implement NTP intro message popup_Existing user - PASSED</summary>


- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.10.20)`. Open an NTP and ensure the `NTP intro onboarding` message is not shown

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and upgrade the profile to `1.33 (21.12.10.20)` and ensure `NTP intro onboarding` message is not shown in the NTP 

</details>

### JP Region

<details>
<summary>Onboarding flow_New user - PASSED</summary>

**Welcome screens**

- Confirmed welcome onboarding flow is shown correctly for JP region
- Confirmed content text of each slide is as per the Figma spec
- Confirmed click on `Skip` button in any slide opens up the NTP
- Confirmed enter any website via last slide opens up the entered website correctly in a NTP

**_Light theme_**
Example | Example | Example | Example 
---------|---------|---------|---------
![Image from iOS (8)](https://user-images.githubusercontent.com/387249/145659703-ab33e47f-ea36-4d1a-ad31-0418ae0ca18d.png) | ![Image from iOS (9)](https://user-images.githubusercontent.com/387249/145659711-0c5305ef-c045-43f7-8b9e-9284aa56e1e9.png) | ![Image from iOS (10)](https://user-images.githubusercontent.com/387249/145659717-e28609b1-cb74-49ff-ac1a-2c408f5782c1.png) | ![Image from iOS (11)](https://user-images.githubusercontent.com/387249/145659722-a1773d14-155e-4a94-b0a4-d5b3fc941a11.png)




**_Dark theme_**
Example | Example | Example | Example 
---------|---------|---------|---------
![Image from iOS (4)](https://user-images.githubusercontent.com/387249/145659731-a45af485-8eeb-4253-97d0-fbe5ec05b602.png) | ![Image from iOS (5)](https://user-images.githubusercontent.com/387249/145659743-fbe74ca7-e989-4b74-bb0b-8ac24d4b24b2.png) | ![Image from iOS (6)](https://user-images.githubusercontent.com/387249/145659752-6494ead0-d7b9-43d2-903d-f684436fe28a.png) | ![Image from iOS (11)](https://user-images.githubusercontent.com/387249/145659760-a9bf71bd-b119-4331-ad8e-e7f2147a7146.png)



**Shield intro message**
- Confirmed shield intro message is shown when any website is opened via NTP
- Confirmed shield intro message popup is shown only once
- Open any website via the last slide and ensure the shield intro message is shown on the opened website
- Tap on any top tiles in NTP and ensure shield intro popup message is shown
- Open an NTP and enter any website and ensure shield intro popup message is shown

Light theme | Dark theme
--------|---------
![Image from iOS (17)](https://user-images.githubusercontent.com/387249/145660196-d7c0b9b0-0316-4dbb-99e0-72f25e31d387.png) | ![Image from iOS (18)](https://user-images.githubusercontent.com/387249/145660198-81960ad3-79d7-4939-9aeb-14198f9ce497.png)


**NTP intro message**
- Confirmed NTP intro message is shown in NTP when welcome onboarding is skipped
- Confirmed NTP intro message is shown only once
- Enter any website via the last slide and ensure shield intro message is shown on the website, ensured an NTP intro message is shown when an NTP is opened
- Tap on any site in the last slide and ensure shield intro message is shown on the website, ensured an NTP intro message is shown when an NTP is opened

**_Light theme_**

Example | Example 
--------|---------
![Image from iOS (15)](https://user-images.githubusercontent.com/387249/145659961-2da73d8d-71fd-4430-b6e9-850ab97229a2.png) | ![Image from iOS](https://user-images.githubusercontent.com/387249/145659967-761f2e7d-97a9-4037-b054-2ed73613f4ac.png)


**_Dark theme_**

Example | Example 
--------|---------
![Image from iOS (1)](https://user-images.githubusercontent.com/387249/145660002-929e5c41-0125-472b-80ee-14efa4e0ce66.png) | ![Image from iOS (3)](https://user-images.githubusercontent.com/387249/145660003-c09e6470-eeb8-4213-abe0-4d2a5b140ebc.png)


</details>

<details>
<summary>Onboarding flow_Existing user - PASSED</summary>

**_Onboarding flow_**
- Install the app store version `1.32.3 (21.11.17.19)` and then upgrade the profile to `1.33 (21.12.10.20)`  and confirm that the welcome onboarding flow isn't shown.
- Install the app store version `1.32.3 (21.11.17.19)` and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.10.20)`  and confirm that the welcome onboarding flow isn't shown.

**_Shield intro_**

-  Install the app store version `1.32.3 (21.11.17.19)` and then upgrade the profile to `1.33 (21.12.10.20)`  and ensure shield intro message is not shown on the active tab, tap on any top tiles in NTP and ensure `shields onboarding intro` isn't shown on the website. 

- Install the app store version `1.32.3 (21.11.17.19)` and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.10.20)` open an NTP and tap on any top tiles and ensure `shields onboarding intro` is not shown on the website 

- Install the app store version `1.32.3 (21.11.17.19)` and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.10.20)` and open a new NTP and enter buzzfeed.com in the URL bar and ensure `shields onboarding intro` is not shown on the website 

- Install the app store version `1.32.3 (21.11.17.19)` and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.10.20)` and long press on any Top tile and open the site in a Private tab and ensure `shields onboarding intro` is not shown on the website

**_NTP intro_**

- Install the app store version `1.32.3 (21.11.17.19)` and upgrade the profile to `1.33 (21.12.10.20)` and ensure the `NTP intro onboarding` message is not shown in the NTP 

</details>

---------
Verification is **PASSED** on `iPhone 8 version iOS 15.1` running `Brave version 1.33 (21.12.13.3)`

### US Region

<details>
<summary>Case 2: Implement welcome screens_Existing user - PASSED</summary>

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and then upgrade the profile to `1.33 (21.12.10.20)`  and confirm that the welcome onboarding flow isn't shown.
- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.10.20)`  and confirm that the welcome onboarding flow isn't shown.
</details>



<details>
<summary>Case 3: Implement shields intro popup_New user_PASSED</summary>

Light theme | Dark theme
-------------|-------------
![IMG_2655](https://user-images.githubusercontent.com/38657976/145839426-3c145bc0-082e-4baa-aaa9-d52e2a737867.PNG) | ![IMG_2661](https://user-images.githubusercontent.com/38657976/145839344-f5dbba17-6459-4fe7-a628-122cf93db035.PNG)

- Clean profile, click on Skip in any welcome screen and open any site via NTP presents the `shields onboarding intro` on the website
- Clean profile `1.33.x`, go through the onboarding flow and select any one website among the 3 in the `You're ready to browse!` slide and confirm that `shields onboarding intro` is shown on the website
- Clean profile `1.33.x`, go through the onboarding flow and enter any website in the `Enter a website` search box in the `You're ready to browse!` slide, and confirm that `shields onboarding intro` is shown on the website
- Clean profile `1.33.x`, skip onboarding flow, enter buzzfed.com in the URL bar and confirm `shields onboarding intro` is shown once on a site. Open the 2nd site and ensure `shields onboarding intro` isn't shown 
- Clean profile `1.33.x`, skip onboarding flow and tap on top tiles in NTP and confirm `shields onboarding intro` is shown on the website
- Clean profile `1.33.x`, skip onboarding flow and long press on any top tiles and open the site in `Private tab` and ensure `shields onboarding intro` is shown on the website and tap outside the bubble and ensure shield onboarding message is dismissed
</details>

<details>
<summary>Case 4: Implement shields intro popup_Existing user-PASSED</summary>

**_User NOT interacted with a shield in 1.32.x_**

-  Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and then upgrade the profile to `1.33 (21.12.6.21)` and tap on any top tiles in NTP and ensure `shields onboarding intro` isn't shown on the website. 

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and then upgrade the profile to `1.33 (21.12.6.21)` and open a new NTP and enter buzzfeed.com in the URL bar and ensure `shields onboarding intro` isn't shown on the website. 

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and then upgrade the profile to `1.33 (21.12.6.21)`  and long press on any Top tile and open the site in a Private tab and ensure `shields onboarding intro` isn't shown on the website

**_User interacted with a shield in 1.32.x_**

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.6.21)` and confirmed `shields onboarding intro` isn't shown on the active tab

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.6.21)` open an NTP and tap on any top tiles and ensure `shields onboarding intro` is not shown on the website 

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.6.21)` and open a new NTP and enter buzzfeed.com in the URL bar and ensure `shields onboarding intro` is not shown on the website 

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.6.21)`  and long press on any Top tile and open the site in a Private tab and ensure `shields onboarding intro` is not shown on the website
</details>

<details>
<summary>Case 5: Implement NTP intro message popup_New user_PASSED</summary>

- Confirmed NTP intro message is shown when a user visits NTP for the 1st time, open 2nd NTP, and ensured the NTP intro message is not shown
- Confirmed content text shown in `NTP intro popup message` is as per the Figma spec
- Clean profile 1.33.x, click on the `Skip` button on any welcome slide opens up NTP with intro message
- Clean profile 1.33.x, click on the `Not now` button in the `Privacy made easy` slide and click on any site in the next slide opens up the website, open a new NTP will show the intro popup intro message
- Clean profile 1.33.x, go through the welcome onboarding flow and click on any website in the `You're ready to browser` slide opens up the website, open a new NTP will show the intro popup intro message
- Clean profile 1.33.x, go through the welcome onboarding flow and enter any website in the `You're ready to browser` slide search box opens up NTP with intro message. Encountered https://github.com/brave/brave-ios/issues/4619

Light theme | Dark theme
-------|--------
![IMG_2659](https://user-images.githubusercontent.com/38657976/145840747-19278460-e80f-4566-b958-972d2206380d.PNG) | ![IMG_2656](https://user-images.githubusercontent.com/38657976/145840811-b8f08b53-93fa-4e65-aaf9-1f79ee783643.PNG)

- Confirmed Tap anywhere outside the intro message popup will dismiss popup in NTP
- Confirmed browser intro more menu `All set!` popup message is shown on NTP after dismissing the NTP intro popup message
- Confirmed content text of the `All set!` popup is as per the Figma spec
- Confirmed Tap anywhere outside the `All set!` popup will dismiss the more menu popup message
- Click on the 3 dot menu opens up the settings page

Light theme | Dark theme
-------|--------
![IMG_2660](https://user-images.githubusercontent.com/38657976/145840928-9d50fe99-215d-4b9f-b6cb-409a56764fe5.PNG) | ![IMG_2657](https://user-images.githubusercontent.com/38657976/145840988-bd736c0b-4747-4c33-b644-81091f550db9.PNG)


</details>

<details>
<summary>Case 6: Implement NTP intro message popup_Existing user_PASSED</summary>

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.11.30.21)`. Open an NTP and ensure the `NTP intro onboarding` message is not shown

- Install the app store version `1.32.3 (21.11.17.19)` click on `Skip/Continue` in NTP and upgrade the profile to `1.33 (21.11.30.21)` and ensure `NTP intro onboarding` message is not shown in the NTP 

</details>

### JP Region

<details>
<summary>Onboarding flow_New user_PASSED</summary>

**Welcome screens**

- Confirmed welcome onboarding flow is shown correctly for JP region
- Confirmed content text of each slide is as per the Figma spec
- Confirmed click on `Skip` button in any slide opens up the NTP
- Confirmed enter any website via last slide opens up the entered website correctly in a NTP

**Shield intro message**
- Confirmed shield intro message is shown when any website is opened via NTP
- Confirmed shield intro message popup is shown only once
- Open any website via the last slide and ensure the shield intro message is shown on the opened website
- Tap on any top tiles in NTP and ensure shield intro popup message is shown
- Open an NTP and enter any website and ensure shield intro popup message is shown

Light theme | Dark theme
--------|---------
![IMG_2664](https://user-images.githubusercontent.com/38657976/145850670-0030c3f1-456b-485c-9c02-23cb392b8051.PNG) | ![IMG_2667](https://user-images.githubusercontent.com/38657976/145850699-dd6463e6-592d-45ef-80a8-5a3229ac3277.PNG)


**NTP intro message**
- Confirmed NTP intro message is shown in NTP when welcome onboarding is skipped
- Confirmed NTP intro message is shown only once
- Enter any website via the last slide and ensure shield intro message is shown on the website, ensured an NTP intro message is shown when an NTP is opened
- Tap on any site in the last slide and ensure shield intro message is shown on the website, ensured an NTP intro message is shown when an NTP is opened

**_Light theme_**

Example | Example 
--------|---------
![IMG_2662](https://user-images.githubusercontent.com/38657976/145851173-e93ce082-e240-4a0a-b51d-c91153e11b27.PNG) | ![IMG_2663](https://user-images.githubusercontent.com/38657976/145851200-fb15de87-b2f7-49c2-b094-2a6672e2e629.PNG)

**_Dark theme_**

Example | Example 
--------|---------
![IMG_2665](https://user-images.githubusercontent.com/38657976/145851254-9eb26c4b-ed25-436a-8926-d240508b2647.PNG) | ![IMG_2666](https://user-images.githubusercontent.com/38657976/145851291-6a1d3f6d-3b4e-402b-acc3-c9b227e41d3b.PNG)

</details>

<details>
<summary>Onboarding flow_Existing user_PASSED</summary>

**_Onboarding flow_**
- Install the app store version `1.32.3 (21.11.17.19)` and then upgrade the profile to `1.33 (21.12.6.21)`  and confirm that the welcome onboarding flow isn't shown.
- Install the app store version `1.32.3 (21.11.17.19)` and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.6.21)`  and confirm that the welcome onboarding flow isn't shown.

**_Shield intro_**

-  Install the app store version `1.32.3 (21.11.17.19)` and then upgrade the profile to `1.33 (21.11.30.21)`  and ensure shield intro message is not shown on the active tab, tap on any top tiles in NTP and ensure `shields onboarding intro` isn't shown on the website. 

- Install the app store version `1.32.3 (21.11.17.19)` and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.6.21)` open an NTP and tap on any top tiles and ensure `shields onboarding intro` is not shown on the website 

- Install the app store version `1.32.3 (21.11.17.19)` and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.6.21)` and open a new NTP and enter buzzfeed.com in the URL bar and ensure `shields onboarding intro` is not shown on the website 

- Install the app store version `1.32.3 (21.11.17.19)` and open a bunch of websites and interact with shield then upgrade the profile to `1.33 (21.12.6.21)` and long press on any Top tile and open the site in a Private tab and ensure `shields onboarding intro` is not shown on the website

**_NTP intro_**

- Install the app store version `1.32.3 (21.11.17.19)` and upgrade the profile to `1.33 (21.12.6.21)` and ensure the `NTP intro onboarding` message is not shown in the NTP 

</details>

**Comment by stephendonner** [comment]
Oops; removing `QA Pass - iPhone` label for now, until @kjozwiak had had a chance to wrap up iOS `15` testing (sorry!)

**Comment by kjozwiak** [comment]
Thanks for running through the remaining checks on `iOS 15` @GeetaSarvadnya 👍 Ran through the `Default Browser` cases and everything is working as expected 👍
